compiler/rustc_codegen_cranelift/src/vtable.rs

```diff
@@ -1,6 +1,6 @@
 //! Codegen vtables and vtable accesses.
 //!
-//! See librustc_codegen_llvm/meth.rs for reference
+//! See `rustc_codegen_llvm/meth.rs` for reference
```

Comment:
This is actually not a correct path, lower similar problems too.

Reply:
Ah! Good catch.

Reply:
Could/should we be linking to the module docs instead of a file path here?

Reply:
I prefer file path as the actual definition of the functions is what matters not the interface that the docs show.

Reply:
In this particular line, the file doesn't exist anymore. Shall we remove this line, or direct readers somewhere else?

Reply:
The new file name is rustc_codegen_ssa/src/meth.rs.

Reply:
Thank you!